vue-js/src/components/task1/List.vue

```diff
@@ -0,0 +1,60 @@
+  <template>
+  <div id="list">
+	<h3>{{headline}}</h3>
+	<ul>
+      <list-item v-for="(item, index) in ToDoList"
+                 v-bind:item='item'
+                 v-bind:key=index
+                 v-on:delete-item="deleteThisItem(index)">
+      </list-item>
+      
+	</ul>
+  <div id="add">
+        <input id="add-item-text" />
+        <button v-on:click="addText">Add</button>
+  </div>
+  </div>
+</template>
+
+<script>
+import ListItem from "./ListItem" 
+
+export default {
+  name: "List",
```

Comment:
```suggestion
  name: "list",
```

Then you can use the import without specifying the tag name.
```
components: {
    List
}
```

See above :point_up: